package.json

```diff
@@ -24,7 +24,6 @@
     "start": "lerna run start --stream --scope docs-app",
     "start:watch": "lerna run start:watch --stream --scope docs-app",
     "dev": "yarn start:watch",
-    "pretest:patchset": "yarn bootstrap && yarn check",
     "test:patchset": "npm-run-all lint:commit lint:changes test:jest test",
     "pretest:merge": "yarn bootstrap && yarn check",
```

Comment:
as I see this is not used anywhere so we should remove it (and `yarn check` is deprecated anyway https://classic.yarnpkg.com/en/docs/cli/check/#toc-yarn-check )

Reply:
I did remove it, for the same reasons you mentioned. :D